test/schema/TlvSchemaTest.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file Unit tests for {@link TlvSchema}.
+ * @copyright Project CHIP Authors 2022
+ * @license Apache-2.0
+ */
+
+import assert from "assert";
+import { UnsignedNumberTlv } from "../../src/schema/TlvSchema";
+import { arrayBufferFromHex, arrayBufferToHex } from "../../src/util/ArrayBuffer";
+
+describe("TlvSchema", () => {
+
+    context("encode", () => {
+        it("encodes an 1 byte unsigned int using in TLV", () => {
+            const result = UnsignedNumberTlv.encode(1);
```

Comment:
Could these tests get simplified into one small test driven by a testVector[] of `{input:, output:}`?
Example:
https://github.com/project-chip/matternode/blob/main/test/tlv/TestTlvCodec.js#L24

Reply:
Will do.